src/olympia/addons/admin.py

```diff
@@ -101,7 +101,7 @@ def clean_path(self):
             raise validation_error
         except Exception:
             raise forms.ValidationError('Path [%s] is not valid' % path)
-        return super(ReplacementAddonForm, self).clean()
```

Comment:
That was causing us to return the whole dict of cleaned data instead of just the one field this method is taking care of...